src/processes/nix_list.rs

```diff
@@ -1,18 +1,23 @@
 use crate::intercept_conf::PID;
 use crate::processes::{ProcessInfo, ProcessList};
 use anyhow::Result;
+
+#[cfg(target_os = "macos")]
 use cocoa::base::nil;
+#[cfg(target_os = "macos")]
 use cocoa::foundation::NSString;
-use core_foundation::number::kCFNumberSInt32Type;
-use core_foundation::number::CFNumberGetValue;
-use core_foundation::number::CFNumberRef;
+#[cfg(target_os = "macos")]
+use core_foundation::number::{kCFNumberSInt32Type, CFNumberGetValue, CFNumberRef};
+#[cfg(target_os = "macos")]
```

Comment:
Thoughts on just moving `visible_windows` to `macos_visible_windows.rs`?
(I don't know what I like better, just bringing it up)

Reply:
I like it - makes the imports cleaner

Reply:
I've taken a slightly different approach, WDYT?